cheroot/ssl/builtin.py

```diff
@@ -172,22 +235,155 @@ def get_environ(self, sock):
             'HTTPS': 'on',
             'SSL_PROTOCOL': cipher[1],
             'SSL_CIPHER': cipher[0],
-            # SSL_VERSION_INTERFACE     string  The mod_ssl program version
-            # SSL_VERSION_LIBRARY   string  The OpenSSL program version
+            'SSL_CIPHER_EXPORT': '',
+            'SSL_CIPHER_USEKEYSIZE': cipher[2],
+            'SSL_VERSION_INTERFACE': 'Python %s' % sys.version,
```

Comment:
This should probably include versions of Cheroot (also available as a server attribute https://github.com/cherrypy/cheroot/blob/38c6338/cheroot/server.py#L1519-L1520) and OpenSSL backend too...

Reply:
I added cheroot but left out openssl since it is already in the environment variable `'SSL_VERSION_LIBRARY'` immediately below